core/cloudflow-spark/src/main/scala/cloudflow/spark/SparkStreamlet.scala

```diff
@@ -230,6 +230,8 @@ trait SparkStreamlet extends Streamlet with Serializable {
  */
 abstract class SparkStreamletLogic(implicit val context: SparkStreamletContext) extends StreamletLogic[SparkStreamletContext] {
 
+  override def getContext(): SparkStreamletContext = super.getContext()
```

Comment:
missed out

Reply:
Cool. Probably never found out cause of missing spark java api..